package.json

```diff
@@ -83,16 +84,16 @@
   "devDependencies": {
     "@storybook/addon-actions": "^4.1.1",
     "@storybook/addon-links": "^4.1.1",
-    "@storybook/react-native": "^4.1.1",
-    "babel-jest": "^23.6.0",
-    "babel-plugin-jest-hoist": "^23.2.0",
+    "@storybook/react-native": "^4.1.14",
+    "babel-jest": "^24.5.0",
+    "babel-plugin-jest-hoist": "^24.3.0",
     "bpk-tokens": "^27.4.7",
     "colors": "^1.3.3",
     "danger": "^7.0.11",
     "eslint-config-skyscanner-with-prettier": "^0.8.0",
     "eslint-plugin-flowtype": "^3.2.0",
     "eslint_d": "^7.2.0",
-    "flow-bin": "^0.78.0",
+    "flow-bin": "^0.92.0",
```

Comment:
To align with RN 0.59's Flow version.

Reply:
Nitpick: maybe this should explicitly be `"0.92.0"`.

Fully appreciate that I probably set the carat last time haha

Reply:
Haha - well I went with the carat because that's [what RN does](https://github.com/facebook/react-native/blob/v0.59.0/package.json#L230) in their repo but I'm happy to lock it if you feel strongly about it.

Reply:
No lets follow them :)